app/assets/javascripts/browsered/worldpay-3ds-flex-ddc.js

```diff
@@ -42,10 +42,15 @@ const checkOriginHostName = origin => {
   return Charge.worldpay_3ds_flex_ddc_url.lastIndexOf(origin, 0) === 0
 }
 
+const updateDdcStatusField = (status) => {
+  document.getElementById('worldpay3dsFlexDdcStatus').value = status
```

Comment:
Wondering if we should call this “status” given DDC thing called “status”. We’ve already used “result” too. Maybe `worldpay3dsFlexDdcReport`? Or maybe it’s fine to leave it as “status”.

Reply:
i reckon it's fine, it won't take people long to figure out what it means based on how we set it